### PR TITLE
update all.json

### PR DESCRIPTION
**Pull request by ical10**
|Site|Urlscan|
|--|--|
|app.dapptokenvalidator.com|https://urlscan.io/result/d54af2f0-5998-49a3-9792-e74aac19e547/dom/|
|block-sync.live|https://urlscan.io/result/0344d4be-85ff-491b-a216-d2f36bf4b1f7/dom/|
|blocksynchronizer.com|https://urlscan.io/result/e03442ca-24c7-4e89-b3ca-5f8dc9e56592/dom/|
|dappintegrates.live|https://urlscan.io/result/eb6aead9-f09c-404b-96fe-570af135120a/|
|defisupport.live|https://urlscan.io/result/68a45894-0f48-48fe-8667-0ac61499e10b/dom/|
|resync4block.com|https://urlscan.io/result/c1840c33-0ad4-446c-a26c-08eb5bae450e/dom/|

app dapptokenvalidator com_SITE
block-sync live_SITE
blocksynchronizer com_SITE
dappintegrates live_SITE
defisupport live_SITE
resync4block com_SITE

### CODE REVIEW

**Comment by ical10** [comment]
ready for review 🚀

File: all.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
 	"allow": [],
 	"deny": [
+		"defisupport.live",
+		"dappintegrates.live",
+		"resync4block.com",
+		"blocksynchronizer.com",
+		"block-sync.live",
+		"app.dapptokenvalidator.com",
```
Comment (jacogr) [inline all.json:9]:
```suggestion
		"app.dapptokenvalidator.com",
		"dapptokenvalidator.com"
```

File: all.json [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+		"dapptokenvalidator.com"
```
Comment (jacogr) [inline all.json:10]:
```suggestion
		"dapptokenvalidator.com",
```

**Review by jacogr — APPROVED** [review]

**Comment by ical10** [comment]
Thanks for the correction Jaco!

**Pull request merged**